pkg/mons/restore_quorum.go

```diff
@@ -327,7 +327,7 @@ func validateMonIsUp(ctx context.Context, clientsets *k8sutil.Clientsets, cluste
 
 	logging.Info("mon %q state is %q", monID, monStatusOut.State)
 
-	if monStatusOut.State == "leader" || monStatusOut.State == "peon" {
+	if monStatusOut.State == "leader" || monStatusOut.State == "peon" || monStatusOut.State == "probing" {
```

Comment:
@subhamkrai try to change this error message in future PR
	
	return fmt.Errorf("mon %q in %q state but must be in leader/peon/probing state", monID, monStatusOut.State)